src/server/common.cc

```diff
@@ -321,4 +325,19 @@ GenericError Context::ReportErrorInternal(GenericError&& err) {
   return err_;
 }
 
+bool AbslParseFlag(std::string_view in, dfly::MaxMemoryFlag* flag, std::string* err) {
+  int64_t val;
+  if (dfly::ParseHumanReadableBytes(in, &val) && val >= 0) {
+    flag->value = val;
+    return true;
+  }
+
+  *err = "Use human-readable format, eg.: 1G, 1GB, 10GB";
```

Comment:
I know you've moved this code, but perhaps a better example would use other units of measurements to show capabilities?
For example "Use human-readable format, eg.: 500M, 10G, 1T".

Reply:
sure